dashboard/ptvsd_util.py

```diff
@@ -0,0 +1,25 @@
+import os, logging
```

Comment:
instead of having this file directly under the `dashboard` can we put this inside folder `debug_util`. This is not core part of project but important tool for development effort. This way somebody who don't know about can find it easy to relate it as debugging

Reply:
Yeah, I wasn't sure where to put this module. I could move it out into it's own directory. I thought about having it in .vscode but I don't believe module names (or directories) can have a period.

Reply:
debug_util sounds as good as any, maybe debug_utils since there might be more!